### PR TITLE
Update golang spdy to new path.

### PR DESCRIPTION
**Pull request by thockin**
Upstream golang spdy was moved to x/net/spdy and then deleted.  This vendors in
the last revision and updates the only user of it.

### CODE REVIEW

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -147,7 +142,7 @@
 		},
 		{
 			"ImportPath": "github.com/docker/spdystream",
-			"Rev": "99515db39d3dad9607e0293f18152f3d59da76dc"
+			"Rev": "83ae67e694a4ab5cbaee4d3126f25118712b26e6"
```
Comment (lavalamp) [inline Godeps/Godeps.json]: Why did this rev change?

**Comment by lavalamp** [comment]
LGTM

File: Godeps/Godeps.json (hunk 1 above)
Comment (thockin) [inline Godeps/Godeps.json]: docker/spdystream changed from code.google.com -> golang.org

**Comment by thockin** [comment]
Doing a fresh `godep restore` after this still gives me:

```
$ godep restore
package golang.org/x/net/spdy
	imports golang.org/x/net/spdy
	imports golang.org/x/net/spdy: cannot find package "golang.org/x/net/spdy" in any of:
	/usr/lib/google-golang/src/golang.org/x/net/spdy (from $GOROOT)
	/tmp/k/src/golang.org/x/net/spdy (from $GOPATH)
godep: restore: exit status 1
```

**Comment by lavalamp** [comment]
In that case, then I recommend moving it to third_party/golang.org/x/net/spdy, updating our import paths accordingly, and removing it from godep, which probably wasn't built for this use case.

**Comment by thockin** [comment]
I don't get how third_party works - it's not in GOPATH, but this package is still depended on by docker/spdystream - godep has a Rev field just for this...

**Comment by lavalamp** [comment]
It's depended on by docker/spdystream? I missed that detail. That makes things hard.

**Comment by lavalamp** [comment]
Filed https://github.com/tools/godep/issues/210... This is probably an uncommon and untested case... :/

**Comment by thockin** [comment]
The hash in our Godeps file should still be git gettable...

**Comment by lavalamp** [comment]
OK, I guess we'll just have to live with the error message for now. LGTM

**Comment by thockin** [comment]
I still don't understand the error, though

**Comment by lavalamp** [comment]
Maybe we should consider switching to https://github.com/constabulary/gb.

**Pull request merged**